view/common/resource-values.phtml

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
I'd like to see the commit message that pulls in this file describe better where this file came from and why we are forking it.

I wonder if our dev docs for this project should also note all files we've forked from the theme as they are the most likely to run into problems when doing version upgrades I imagine.

Reply:
For context on the number of template files in this theme, and how many have been forked - of the 22 .phtml files in the theme as of this PR, only 7 are not forked from the core application.

Reply:
The updated commit message is very helpful thanks.

Now that I understand that _most_ `.phtml` templates are forked, maybe our docs just state that _most_ files are forked (with a link to where they came from) and maybe noting which once we introduced ourselves? That goes beyond this change though so feel free to ignore for now and consider it as part of future maintenance. It's very much not-essential, just a thought fwiw.